examples/sglang/utils/wideep/sgl_bench_one_batch_server.py

```
@@ -0,0 +1,380 @@
+
+    for chunk in response.iter_lines(decode_unicode=False):
+        chunk = chunk.decode("utf-8")
+        if chunk.startswith("data: [DONE]"):
+            break
+        if not chunk.startswith("data:"):
+            continue
+
+        data_str = chunk[len("data:"):].strip()
+        try:
+            data = json.loads(data_str)
+        except Exception as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)